packages/story-utils/src/generateSnapshots.ts

```diff
@@ -53,23 +53,23 @@ export default function generateSnapshots(
     test(`${storyName} story renders snapshot`, async () => {
       const view = render(createElement(Story));
 
+      // When components that include Apollo's useQuery are rendered we need
+      // to await an act that pushes the test to the end of the event loop.
+      // https://trojanowski.dev/apollo-hooks-testing-without-act-warnings/
+      await wait();
```

Comment:
Moved the `wait` before running play functions. That way we're more sure there's something actually rendered when play functions run.